ctyparser/bigcty.py

```diff
@@ -63,17 +48,20 @@ class BigCty(collections.abc.Mapping):
                                  (?:~(?P<tz>[+-]?\d+(?:\.\d+)?)~)?""", re.X)
 
     def __init__(self, file_path: Union[str, os.PathLike, None] = None):
+        """Can be initialised with data by passing the path to a valid ``cty.json`` file to the constructor.
+        """
```

Comment:
```suggestion
```
Docstrings are superfluous for `__init__()`.

Reply:
fixed